Docker/README.md

```diff
@@ -49,6 +49,13 @@ Start 3 MPI nodes:
 ```bash
     source docker-compose-openmpi.sh up --scale mpi_head=1 --scale mpi_node=3
 ```
+
+**Testing OpenMPI with optional custom ssh_key for Borg** 
+
+Note that without specifying a location, it will assume the ssh for bitbucket is set up in the standard location (e.g. `~/.ssh/id_rsa`)
+```bash
+    BITBUCKET_SSH_KEY=/.../id_rsa source docker-compose-openmpi.sh up --scale mpi_head=1 --scale mpi_node=3
```

Comment:
what is `/.../`?

Reply:
Ah, I was trying to write an arbitrary location, how would I do so?

Reply:
Sorry, thought I replied to this, but apparently I didn't.  Maybe something like `/path/to/id_rsa` or `<path_prefix>/id_rsa`. Would be good to know if there's a standard, but if it exists I don't know it.